src/typings.ts

```diff
@@ -17,6 +22,7 @@ export interface ConfigsOptions extends PrivateConfigsOptions {
   showRememberMe?: boolean;
   isNewTab?: boolean;
   forceLogout?: boolean;
+  authnMethod?: AuthnMethod | AuthnMethod[];
```

Comment:
Since `authnMethod` is in `ConfigsOptions` this means it will need to be passed to every API method, right?

@mt-max What happens if we pass it to `authorize` but later not to `openService`?